dist/tools/ethos/start_network.sh

```diff
@@ -2,31 +2,14 @@
 
 ETHOS_DIR="$(dirname $(readlink -f $0))"
 
-create_tap() {
-    ip tuntap add ${TAP} mode tap user ${USER}
-    sysctl -w net.ipv6.conf.${TAP}.forwarding=1
-    sysctl -w net.ipv6.conf.${TAP}.accept_ra=0
-    ip link set ${TAP} up
-    ip a a fe80::1/64 dev ${TAP}
-    ip a a fd00:dead:beef::1/128 dev lo
-    ip route add ${PREFIX} via fe80::2 dev ${TAP}
-}
-
-remove_tap() {
-    ip tuntap del ${TAP} mode tap
-}
-
 cleanup() {
-    echo "Cleaning up..."
-    remove_tap
-    ip a d fd00:dead:beef::1/128 dev lo
-    kill ${UHCPD_PID}
+    [ -n "${SETUP_NETWORK_PID}" ] && kill -TERM ${SETUP_NETWORK_PID}
     trap "" INT QUIT TERM EXIT
 }
 
-start_uhcpd() {
-    ${UHCPD} ${TAP} ${PREFIX} > /dev/null &
-    UHCPD_PID=$!
+setup_network() {
+    ${ETHOS_DIR}/setup_network.sh $1 $2 &
```

Comment:
In the spirit of future proofness:
```suggestion
    ${ETHOS_DIR}/setup_network.sh $* &
```

So later extensions to the script do not need this function to be touched.